account_invoice_blocking/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Account Invoice Blocking
+========================
+
+This module allows the user to set a blocking (No Follow-up) flag directly on the invoice. This facilitates the blocking of the invoice's move lines.
+
+Usage
+=====
+
```

Comment:
As better you describe how to use the module, as much more easier is to test it, so more people will be able to review it. It's just a recommendation. I rebuilding runbot for testing